BREEZE.rst

```diff
@@ -46,409 +46,552 @@ Here is the short 10 minute video about Airflow Breeze
+- **Disk space**: On macOS, increase your available disk space before starting to work with the environment. At least 128 GB of free disk space is recommended. You can also get by with a smaller space but make sure to clean up the Docker disk space periodically. See also `Docker for Mac - Space <https://docs.docker.com/docker-for-mac/space>`_ for details on increasing disk space available for Docker on Mac.
 
-When you develop on Mac OS you usually have not enough disk space for Docker if you start using it
-seriously. You should increase disk space available before starting to work with the environment.
-Usually you have weird problems of docker containers when you run out of Disk space. It might not be
-obvious that space is an issue. At least 128 GB of Disk space is recommended. You can also get by with smaller space but you should more
-often clean the docker disk space periodically.
+  Sometimes it is not obvious that space is an issue when you run into a problem with Docker. If you see a weird behaviour, try `cleaning up the images <#cleaning-up-the-images>`_.
 
-If you get into weird behaviour try `Cleaning up the images <#cleaning-up-the-images>`_.
+Docker Compose
+--------------
 
-See also `Docker for Mac - Space <https://docs.docker.com/docker-for-mac/space>`_ for details of increasing
-disk space available for Docker on Mac.
+- **Version**: Install the latest stable Docker Compose and add it to the PATH. See `Docker Compose Installation Guide <https://docs.docker.com/compose/install/>`_ for details.
 
-Docker compose
---------------
+- **Permissions**: Configure to run the ``docker-compose`` command.
 
-Latest stable Docker Compose installed and on the PATH. It should be
-configured to be able to run ``docker-compose`` command.
-See `Docker compose installation guide <https://docs.docker.com/compose/install/>`_
+Docker Images Used by Breeze
+----------------------------
 
-Getopt and gstat
-----------------
+For all development tasks, related integration tests and static code checks, we use Docker
+images maintained on the Docker Hub in the ``apache/airflow`` repository.
 
-* If you are on MacOS
+There are three images that we are currently managing:
 
-  * you need gnu ``getopt`` and ``gstat`` to get Airflow Breeze running.
+* **Slim CI** image that is used for static code checks (size of ~500MB). Its tag follows the pattern
+  of ``<BRANCH>-python<PYTHON_VERSION>-ci-slim`` (for example, ``apache/airflow:master-python3.6-ci-slim``).
+  The image is built using the `<Dockerfile>`_ Dockerfile.
+* **Full CI image*** that is used for testing. It contains a lot more test-related installed software
+  (size of ~1GB). Its tag follows the pattern of ``<BRANCH>-python<PYTHON_VERSION>-ci``
+  (for example, ``apache/airflow:master-python3.6-ci``). The image is built using the
+  `<Dockerfile>`_ Dockerfile.
+* **Checklicence image** that is used during license check with the Apache RAT tool. It does not
```

Comment:
Is it `license` or `licence` ? Should we be consistent with that?

Reply:
I adjusted all the names to license :). Still there are some variables etc with LICENCE. I will adapt it in production image change.